pkg/daemon/update.go

```diff
@@ -306,7 +306,17 @@ func ExtractOSImage(imgURL string) (osImageContentDir string, err error) {
 	return
 }
 
-func (dn *Daemon) applyOSChanges(oldConfig, newConfig *mcfgv1.MachineConfig) error {
+// Remove pending deployment on OSTree based system
+func removePendingDeployment() error {
+	args := []string{"cleanup", "-p"}
+	_, err := runGetOut("rpm-ostree", args...)
+	if err != nil {
+		return err
+	}
+	return nil
```

Comment:
nit you can `return err` directly if you don't plan to wrap it

Reply:
right, fixed.